log.go

```diff
@@ -78,8 +93,61 @@ func (hook *logrusFileHook) Levels() []log.Level {
 	}
 }
 
+// StartWritingStats writes fm.Stats every minute to Config.StatsFile
+func (fm *Frontman) StartWritingStats() {
```

Comment:
Maybe put comment this function can be invoked only once or better wrap it to `sync.Once`

Reply:
I've added a comment. I don't think wrapping `sync.Once`  around is necessary. It's no library method and unlikely to call it by accident I think.